### ISSUE TITLE
NotFoundError(see above for traceback):Tensor name ''InceptionV1/Mixed_4b/Branch_1/Conv2d_0a_1*1/biases" not found in checkpoint file inception_v1.ckpt''

### ISSUE THREAD

**Issue by zzzl-h** [tensorflow/models#3626]
My code
slim = tf.contrib.slim

x=tf.placeholder("float",[None,224,224,3])
y=tf.placeholder("float",[None,num_classes])

logits, collection= inception.inception_v1(x,num_classes=num_classes)
predictions = collection['Predictions']
loss=-tf.reduce_sum(y*tf.log(predictions))
optimizer=tf.train.GradientDescentOptimizer(learning_rate=learning_rate).minimize(loss)
accuracy=tf.reduce_mean(tf.cast(tf.equal(tf.argmax(predictions, 1), tf.argmax(y, 1)),"float"))

saver=tf.train.Saver()
with tf.Session() as sess:
    saver.restore(sess,"inception_v1.ckpt")

**Comment by Shirhe-Lyh** [comment]
@zzzl-h  Solution: when you calling the function assign_from_checkpoint_fn, you should set ignore_missing_vars=True like this:

init_fn = slim.assign_from_checkpoint_fn(checkpoint_path,
variables_to_restore,
ignore_missing_vars=True)

**Comment by mefathy** [comment]
@Shirhe-Lyh I guess the issue here is that the checkpoint file provided in this repo for the inception_v1 is invalid. If you inspect the variables defined inside the provided checkpoint file (inception_v1.ckpt), you will find variables for batch normalization layers, which are not part of the inception_v1 model, and you will not find any bias variables.

@zzzl-h The best thing to do for now is to switch to another model or retrain it inception_v1 yourself from scratch. Any result you will get by setting ignore_missing_vars=True with the current setting may not reflect the actual performance of an inception_v1 model pretrained on image-net.

**Comment by wt-huang** [comment]
Closing as this is resolved
